src/cascadia/TerminalSettingsModel/KeyChordSerialization.cpp

```diff
@@ -104,14 +105,12 @@ static const std::unordered_map<std::wstring_view, int32_t> vkeyNamePairs {
 // - hstr: the string to parse into a keychord.
 // Return Value:
 // - a newly constructed KeyChord
-KeyChord KeyChordSerialization::FromString(const winrt::hstring& hstr)
+static KeyChord _FromString(const std::wstring_view& wstr)
 {
-    std::wstring wstr{ hstr };
-
     // Split the string on '+'
     std::wstring temp;
     std::vector<std::wstring> parts;
-    std::wstringstream wss(wstr);
+    std::wstringstream wss(wstr.data());

```

Comment:
huh, wstringstream cannot consume a view?

Reply:
No. 😭

Reply:
Oh I totally missed that... This is a bug!
`wstring_view` doesn't need to have a terminal null character!

Please write the following instead:
```cpp
std::wstringstream wss;
wss << wstr;
```